ui/semantic/tasks/watch.js

```diff
@@ -0,0 +1,231 @@
+/*******************************
+           Watch Task
+*******************************/
+
+var
+  gulp         = require('gulp-help')(require('gulp')),
```

Comment:
END IGNORE PULL REQUEST. GitHub freaking sucks for this. Ultimately, we'll have to do a code review over Zoom. Stupid Semantic folder.